apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -42,6 +42,10 @@ export default class ProjectTask implements ITaskDefinition {
     return this._rushProject.packageName;
   }
 
+  public get logName(): string {
```

Comment:
>logName [](start = 13, length = 7)

This should be calculated once, probably by RushConfigurationProject.  We definitely shouldn't be redoing the formatting  every time we print.